app/assets/javascripts/shared/AnalyticsService.js.coffee

```diff
@@ -6,27 +6,38 @@ AnalyticsService = ($state) ->
   Service = {}
   Service.timer = {}
 
+  Service.resetProperties = {
```

Comment:
Should this object have any values?

Reply:
Not currently, there may be some values we want to nullify in the future though so I left the empty object in there